<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{border:1px solid #92C8E8; margin: 0px;background-color:white} QPushButton{border:0px;margin:0px;}
QLabel{border:0px;margin:0px;}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/images/upleft</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minBtn">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../widget.qrc">
          <normaloff>:/resources/images/mininum</normaloff>:/resources/images/mininum</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../widget.qrc">
          <normaloff>:/resources/images/close</normaloff>:/resources/images/close</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="styleSheet">
      <string notr="true">border:1px solid #92C8E8; background-color:#D6EFFB;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="bottomLayout">
        <item>
         <widget class="QWidget" name="bottomCenterWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>290</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{border:0px;}
QLabel{color:#3F3A39;font-size:16px;font-family:楷体;}
QLineEdit{border:1px solid #CBD2D5 ;height:32px;border-radius:6px;background:white;padding-left:5px;}
QPushButton{border:0px;background:#F3A026;color:white;border-radius:6px;font-size:16px}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="nicknameLayout">
             <item>
              <widget class="QLabel" name="nicknameLabel">
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>昵称</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="nicknameLineEdit">
               <property name="maximumSize">
                <size>
                 <width>230</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="pwdLayout">
             <item>
              <widget class="QLabel" name="passwdLabel">
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>密码</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwdLineEdit">
               <property name="maximumSize">
                <size>
                 <width>230</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="confirmLayout">
             <item>
              <widget class="QLabel" name="confirmLabel">
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>确认密码</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="confirmLineEdit">
               <property name="maximumSize">
                <size>
                 <width>230</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="registerLayout">
             <item>
              <widget class="QPushButton" name="registerBtn">
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>38</height>
                </size>
               </property>
               <property name="text">
                <string>注册</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="returnLayout">
             <item alignment="Qt::AlignRight|Qt::AlignVCenter">
              <widget class="QPushButton" name="returnBtn">
               <property name="styleSheet">
                <string notr="true">QPushButton{border:0;background-color:transparent;color:#3F3A39;font-size:14px}QPushButton:hover{color:#CBD2D5}</string>
               </property>
               <property name="text">
                <string>返回</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../widget.qrc"/>
 </resources>
 <connections/>
</ui>
